lib/ember-cli/middleware.rb

```diff
@@ -17,6 +19,10 @@ def call(env)
 
     private
 
+    def skip_middleware?
+      %r{/api/}.match(env["REQUEST_URI"])
```

Comment:
@rwz @rondale-sc ideally, this would be configurable.

I envision passing a block to the middleware:

```ruby
config.use EmberCLI::Middleware do |env|
  # still not sure if this should return `true` or `false` (i.e. enable or disable)
  %r{/api/}.match(env["REQUEST_URI"]).nil?
end
```

This type of change would require being explicit, though, so we couldn't insert the middleware by default.

Does Rails gracefully handle the same middleware being declared multiple times?